### DISCUSSION TITLE
How to change the white-balance of a raw image?

### DISCUSSION THREAD

**Discussion by xeggbee** [colour-science/colour#1014] [Q&A] [answered]
Hello, please,
how to change the white balance of raw image?

**Comment by thomasmansencal** [comment, marked as answer]
Hi @xeggbee,

At its most basics, you only really need to gain the different RGB channels in Camera RGB (Camera Native) space. However, it is much more complicated than that as white-balancing needs to be performed before demosaicing.

We have a few examples here but it is a bit involved: https://github.com/colour-science/colour-hdri/tree/develop/colour_hdri/examples

Cheers,

Thomas
